kelle_test.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+from pyspextool.mc_bitset import bitset
+
+
+mask = bitset(np.array([3,4,1]),[0])
```

Comment:
```suggestion
mask = bitset(np.array([1,2,3]),[0])
```
There was a typo in the function and I fixed it.